test/concat.test.js

```diff
@@ -68,6 +68,26 @@ describe('concat', () => {
     strictEqual(result, 'abcd')
   })
 
+  it('should forward error events', async () => {
+    const stream0 = new Readable({
+      read: () => {
+        stream0.destroy(new Error('test'))
```

Comment:
This would be also equivalent to `stream0.emit('error', new Error())`?

Reply:
I think it's a little bit more robust. It will set the destroyed flag, which stops any further processing. Because of the async nature of events, just emitting the event may produce follow up errors.